packages/opentelemetry-core/src/context/propagation/JaegerHttpTraceFormat.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SpanContext, HttpTextFormat, TraceFlags } from '@opentelemetry/types';
+
+export const UBER_TRACE_ID_HEADER = 'uber-trace-id';
```

Comment:
I think that we should be able to override this string with an env variable. WDYT ?

Reply:
@OlivierAlbertini i was working with jaeger tracing and didn't see any another header for HTTP. I think this format only specific to jaeger otherwise we should make it overridable

Reply:
I believe that many companies (that use Jaeger) didn't keep `uber-trace-id` but something like `mycompany-trace-id` so by making it overridable, it will be more reusable.

Reply:
I would make it a constructor option rather than an ENV var. none of our other packages are using ENV for config

Reply:
Make sense. Perhaps we should open a discussion on this aspect because I feel that Cloud Provider would prefer ENV variables than passing configs (for some configs). I'm not a devOps but I guess that this is something more convenient to have ENV variables to set in a jenkinsfile/helm chart etc... WDYT ? I think you have more insight than me on that.

Reply:
Configuration as such has been talked about in spec meetings recently. It's something they want to be consistent across SIGs. I think we're better off waiting for that.

Reply:
DotSpy, feel free to add constructor option or we can handle this in a separate PR as well.